### PR TITLE
#2635 Add ability to perform click at disabled elements

### PR DESCRIPTION
**Pull request by asolntsev**

### CODE REVIEW

**Review by BorisOsipov — COMMENTED** [review]
.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * #2550 Implement downloading files via CDP or BiDi (#2567) - thanks to Sergey Brit!
 * #2612 don't set page load timeout in mobile tests (#2628)
 * #2617 User can safely add the same proxy filter many times (#2630)
+* #2635 Add ability to perform click at disabled elements (#2636)
```
Comment (BorisOsipov) [inline CHANGELOG.md:7]:
```suggestion
* #2635 Add the ability to skip checks before clicking an element (#2636)
```

File: src/main/java/com/codeborne/selenide/ClickOptions.java [hunk 2]
```diff
@@ -109,7 +116,16 @@ public ClickOptions offset(int offsetX, int offsetY) {
...
-    return new ClickOptions(clickMethod, offsetX, offsetY, timeout);
+    return new ClickOptions(clickMethod, offsetX, offsetY, timeout, force);
+  }
+
+  /**
+   * @since 7.1.0
+   */
+  @CheckReturnValue
+  @Nonnull
+  public ClickOptions allowDisabled() {
```
Comment (BorisOsipov) [inline src/main/java/com/codeborne/selenide/ClickOptions.java:127]:
```suggestion
  public ClickOptions force() {
```

File: statics/src/test/java/integration/ClickDisabledButtonTest.java [hunk 3]
```diff
@@ -18,9 +20,23 @@ void failToClickIfButtonIsDisabled() {
       .hasMessageStartingWith("Element should be clickable: interactable and enabled {#submit}");
   }
 
+  @Test
+  void clickingDisabledButtonMayBeAllowed() {
+    assertThatNoException().isThrownBy(() ->
+      $("#submit").click(usingDefaultMethod().allowDisabled())
```
Comment (BorisOsipov) [inline statics/src/test/java/integration/ClickDisabledButtonTest.java:26]:
```suggestion
      $("#submit").click(usingDefaultMethod().force())
```

File: statics/src/test/java/integration/ClickDisabledButtonTest.java [hunk 4]
```diff
@@ -18,9 +20,23 @@ void failToClickIfButtonIsDisabled() {
...
   @Test
   void failToDoubleClickIfButtonIsDisabled() {
     assertThatThrownBy(() -> $("#submit").doubleClick())
       .hasMessageStartingWith("Element should be clickable: interactable and enabled {#submit}");
   }
+
+  @Test
+  void doubleClickingDisabledButtonMayBeAllowed() {
+    assertThatNoException().isThrownBy(() ->
+      $("#submit").doubleClick(usingDefaultMethod().allowDisabled())
```
Comment (BorisOsipov) [inline statics/src/test/java/integration/ClickDisabledButtonTest.java:39]:
```suggestion
      $("#submit").doubleClick(usingDefaultMethod().force())
```

**Review by BorisOsipov — APPROVED** [review]

**Review by rosolko — APPROVED** [review]

**Pull request merged**